src/content/blog/astro-ecosystem-fund-update.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "The $100,000 Astro Ecosystem Fund Fall Update"
+description: "Astro — the web framework for building content-driven websites like blogs, marketing, and e-commerce — is giving away $100,000 to support other open-source projects in our ecosystem that improve the lives of Astro users everywhere, every day. Here's an update on the program, including two new grants."
+publishDate: "September 16, 2024"
+homepageLink:
+  title: "$100,000 Ecosystem Fund"
+  subtitle: "Meet our newest recipients"
+authors:
+  - fred
+coverImage: "/src/content/blog/_images/astro-ecosystem-fund-update/blog-post-community-grants.png"
+socialImage: "/src/content/blog/_images/astro-ecosystem-fund-update/og-community-grants.png"
+lang: "en"
+---
+
+**We pledged **$100,000** to open source maintainers this year to support the projects that most improve the lives of Astro users. We have a new batch of grants to announce today, along with an update on our progress and the state of open source funding across our industry as a whole.**
+
+Astro is the web framework for building content-driven websites like blogs, marketing, and e-commerce. This year we created the Astro Ecosystem Fund to award $100,000 in large grants to ten project maintainers over the course of the year. Each recipient receives a one-time grant with no strings attached, to support and continue their work on behalf of the larger Astro ecosystem. 
+
+I'm excited to announce two new recipients in our latest fall batch:
+- **[@hippotastic](https://github.com/hippotastic)** ($10,000) for work on the [Expressive Code](https://expressive-code.com/) advanced syntax highlighter.
+- **[@martrapp](https://github.com/martrapp)** ($5,000) for work on [VTBOT](https://github.com/martrapp/astro-vtbot) exploring the bleeding edge of View Transitions in Astro and the larger web platform.
+
+We are thankful to our own major open-source sponsors for making this possible including [Netlify](https://www.netlify.com/), [Sentry](https://www.sentry.com/), [Jetbrains](https://www.jetbrains.com/) [and more](https://github.com/withastro/astro#sponsors). Without their support, this initiative would never have been possible.
+
+## State of OSS Funding
+
+Hippo and Martin are the fifth and sixth grant recipients from the fund. They join the maintainers of [Lucia](https://lucia-auth.com/), [Volar](https://volarjs.dev/), [`astro-eslint-parser`](https://github.com/ota-meshi/astro-eslint-parser) and [Lexington themes](https://lexingtonthemes.com/) who all received grants earlier this year. We have $45,000 remaining in the fund, which leaves us with at least 4 more grants to give out before the end of the year. We're already preparing for an exciting final round of announcements!
+
+Since we announced this program last December, we've also seen more attention than ever shift to the unbalanced relationship between free, open source software and the closed, for-profit corporations that benefit the most from them. 
```

Comment:
```suggestion
Since we announced this program last December, we've also seen more attention than ever shift to the unbalanced relationship between free, open-source software and the closed, for-profit corporations that benefit the most from it. 
```